### ISSUE TITLE
Just a suggestion.  Might be worth adding to the project?

### ISSUE THREAD

**Issue by visualjeff** [jelhub/scimgateway#12]
You can debug a running docker container (using Visual Studio Code) if you add the following file (docker-compose-**debug**.yml) to your project:

```
version: '2'
services:
  scimgateway:
    ports:
      - "8880:8880"
      - "5858:5858"
```

When you start-up the app using this command:

  `docker-compose -f docker-compose.yml -f docker-compose-debug.yml up -d
`
Start Visual Studio Code and follow these debugging instructions:  [https://code.visualstudio.com/docs/nodejs/nodejs-debugging](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)

Jeff

PS. Hopefully users might find this helpful.

**Comment by jelhub** [comment]
Will be added to the next release.
Thanks,
Jarle

**Comment by jelhub** [comment]
Hi,
It's now implemented in v0.5.0

Thanks,
Jarle
